cypress/fixtures/tequila_list.json

```diff
@@ -0,0 +1,64 @@
+{
+  "ProductSearchResults": [
+    {
+      "product_id": "19354179",
+      "product_number": "4051201",
+      "product_name": "Don Abraham",
+      "product_name_2": null,
+      "product_category": "Sprit",
+      "product_producer": "Tequilas las Americas",
+      "product_price": 428.0,
+      "product_country": "Mexiko",
+      "product_type": "Tequila",
+      "product_volume": 700
```

Comment:
I would omit the word product in all keys

Reply:
The backend is using product_ in front of the items so that's why we've used it here in the client, should we change this on the backend?

Reply:
Yeah, I would change that across the line.

Reply:
Updating

Reply:
All updated